test/mocks/handlers/index.js

```diff
@@ -1,7 +1,5 @@
-import {authHandlers} from './auth.handler';
+import { authHandlers } from './auth.handler';
 
-const handlers = [
-    ...authHandlers
-]
+const handlers = [...authHandlers];
 
-export default handlers;
\ No newline at end of file
+export default handlers;
```

Comment:
You need to import the mock handler if you have defined it otherwise it could be for nothing.